docs/documentation/full-text/filtering.mdx

```diff
@@ -2,37 +2,62 @@
 title: Filtering
 ---
 
-The ParadeQL query accepts filters, which can drastically improve query times compared to standard SQL `WHERE` clauses.
-For optimal performance and ranking, it is important to pass filters directly into the search query whenever possible.
+Filters over numeric, boolean, or datetime fields that are indexed as [fast fields](/documentation/indexing/fast_fields) can be "pushed down" into the BM25 index using the `@@@` operator. For instance, the following
+two queries produce identical results, but the second query "pushes down" the `rating > 2` filter into the BM25 index.
+
+```sql
+-- Without BM25 pushdown
+SELECT description, rating, category
+FROM mock_items
+WHERE description @@@ 'shoes' AND rating > 2;
+
+-- With BM25 pushdown
+SELECT description, rating, category
+FROM mock_items
+WHERE description @@@ 'shoes' AND rating @@@ '>2';
+```
+
+<Note>
+  Automatically executing the first type of query as the second query is on the
```

Comment:
I think @eeeebbbbrrrr is building this right now. Might even be able to get it in before this gets promoted :)